css-canvas-medical/styles.css

```diff
@@ -0,0 +1,278 @@
+.container {
+  width: 75%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
These are some big augments to the default way a `row` class should operate. I would make this a separate class named something like `flex-column-center` that you can add in addition to `row` on the necessary elements. This will keep `row` unpolluted so you can use it normally elsewhere, if needed.